inc/sysrepo.h

```diff
@@ -158,6 +187,9 @@ typedef struct sr_val_s {
  * than to an actual xpath.
  */
 typedef struct sr_node_s {
+    /** Memory context used by this node. */
+    sr_mem_ctx_t *sr_mem;
```

Comment:
same as above - I would name this as `_sr_mem` (with a `_` in the front) and explicitly state that this is an internal member (as in the .proto file)